### ISSUE TITLE
Login popup should contain example userid & password

### ISSUE THREAD

**Issue by ihh** [GMOD/jblast-jbconnect-hook#226]
The demo at http://graingenes.org:1337/jbrowse/?data=IWGSC no longer has a separate login page, and the popup that appears when you click on the key icon (to login) does not have the example username and password on it.

**Comment by enuggetry** [comment]
The message was in http://graingenes.org:1337/login.
I will add it to the popup too.

**Comment by ihh** [comment]
thanks!

**Comment by enuggetry** [comment]
resolved

**Comment by ihh** [comment]
The popup that appears when you click on the key icon (to login) should have the example username and password on it.

**Comment by enuggetry** [comment]
inserting defaults in the user/pass fields might interfere with the browser's autofill.

**Comment by enuggetry** [comment]
fulfilled by https://github.com/GMOD/jbconnect/issues/102

**Comment by ihh** [comment]
What I had in mind was not inserting defaults in the username or password field, but including some help text in the popup that appears when you click the key to log in. This is the same way that it works in the main login page.

**Comment by enuggetry** [comment]
is it necessary given that the credentials are displayed in the info popup?

**Comment by ihh** [comment]
Yes, it's necessary. A suggestion for how to fill in an input field should be next to the input field where the suggestion is meant to be typed. Not in a separate popup accessed through a different button. We did it this way for the main login page as well (following a similarly protracted discussion of the rationale for why it should be this way.)

Once you've added it to the login popup, it doesn't need to be in the info popup at all, so you can remove it from there.

As an aside, the info popup, while helpful in that it contains a link to documentation, is annoying if you're already logged in or know what you're doing. Ideally it should only appear once, and also it should disappear if you click outside it (rather than just via the close button or the timeout). But this is a minor issue. I guess I'll file it as an issue anyway: #236

**Comment by enuggetry** [comment]
I just think it's a little unconventional to have the credentials show in the application at all.
It might be better to put it in the landing page that was built for this http://jblastdemo...? 
(btw, landing page doesn't go there yet).  
Do we still want a landing page?

**Comment by ihh** [comment]
Displaying suggestions for text inputs is not unconventional at all.
Displaying suggested credentials, for a text input in the login popup that the user can open from within the application, is no more unconventional than displaying those suggested credentials in a login page that the user sees _prior_ to reaching the application. In fact, if you show them on one login interface then you should show them on all login interfaces.
Having a guest login with credentials that the user must read and then manually re-enter is mildly unconventional, and a little clunky. So in this sense, showing suggested credentials is indeed clunky. It would be better to have a "Login as guest" button, so that the user didn't have to type anything. To do this properly you also ought to ensure that the user cannot _change_ the guest password while logged in, nor change any other aspects of the guest account configuration. I avoided specifying this as an issue due to the already extensive list of things to do, but this would be the better path.

The bottom line is that logging in is a complete distraction for a casual user trying out the app for the first time, and it should be made as easy as possible and to require as little attention as possible. Hence, "simple guest login button" is better than "demo credentials displayed next to username/password input fields" which, in turn, is better than "demo credentials displayed in a separate help popup".

**Comment by ihh** [comment]
Hmmm, actually on second thoughts I don't think the landing page is necessary now we have a login popup inside the app.

**Comment by enuggetry** [comment]
display help credentials.  addressed here: https://github.com/GMOD/jbconnect/issues/103
